00-intro.md

```diff
@@ -0,0 +1,41 @@
+# 00. Develop Accessible Web Apps with React
+
+**[📹 Video](https://egghead.io/lessons/react-v2-00-a-beginners-guide-to-react-introduction?pl=a-beginners-guide-to-react-v2-6c4d)**
+
+**[💻 Course Repo](https://github.com/erin-doyle/egghead-react-a11y/tree/labels)**
+
+
+## Notes on Notes 🤓
+
+The word accessibility is often abbreviated to **a11y** with the number eleven in the middle (this refers to the number of letters that the word contains between the first and last letter - a numeronym).
```

Comment:
How do you pronounce a11y? sinc;e you go into a little explanation maybe your input on that would be helpful as well. I'm actually not sure how most people say it. I've said 'alē' (allie) - vs ˈa-ˌlī' (ally) - though I've heard several use the latter.

Reply:
Yep, good idea! 👍